### ISSUE TITLE
Add nvenc video encoder

### ISSUE THREAD

**Issue by jpnielsen** [VCVRack/VCV-Recorder#46]
It has disappointed me, that i was unable to encode fullscreen 4K video / use ASIO driver when encoding.

It seems to me, that including the NVIDIA nvenc in VCV-Recorder would enable this - atleast on Windows 11 and Linux(- ASIO).

https://developer.nvidia.com/nvidia-video-codec-sdk

thank you

Rack2Pro userId: 17189

**Comment by AndrewBelt** [comment]
I don't think it would be possible to distribute VCV Recorder with nvenc due to it being proprietary software in a GPLv3 plugin, and because linking to it would prevent users from loading it unless they have an Nvidia GPU with nvenc drivers installed.

You might be able to add nvenc to the source/Makefile and build it yourself, but I'm not sure how hard that would be.
